bindgen/callbacks.rs

```diff
@@ -211,3 +224,27 @@ pub struct FieldInfo<'a> {
     /// The name of the field.
     pub field_name: &'a str,
 }
+
+/// A struct providing information about the funktion-like macro being passed to [`ParseCallbacks::fn_macro`].
```

Comment:
German and English probably have different ideas about this spelling:
```suggestion
/// A struct providing information about the function-like macro being passed to [`ParseCallbacks::fn_macro`].
```

Reply:
Yes, *probably*. Good catch. 😄